content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

```diff
@@ -0,0 +1,94 @@
+
+This guide describes how to use the Datadog Continuous Profiler to investigate these kinds of problems.
+
+## CPU bursts
+
+The first step in a performance investigation is to identify anomalies in resource usage over time. Consider the following graph of CPU utilization over the past hour for the service `product-recommendation`:
+
+{{< img src="profiler/guide-monolithic-outliers/1-outliers-monolith-cpu-usage.png" alt="" style="width:100%;" >}}
+
+This doesn't provide concrete pointers to the answers, but you can see some anomalous peaks in CPU usage. 
+
+Select the **Show - Avg of** dropdown (highlighted) and change the graph to show `CPU Cores for Top Endpoints` instead. This graph shows how different parts of the application contribute differently to the overall CPU utilization:
+
+{{< img src="profiler/guide-monolithic-outliers/2-outliers-monolith-cpu-top-endpoints.png" alt="" style="width:100%;" >}}
+
+
+The yellow peaks show that the `GET /store_history` endpoint has some intermittent usage corresponding to the anomalies identified above. However, the peaks might be due to differences in traffic to that endpoint. To understand if profiles can provide further insights, change the metric to `CPU - Average Time Per Call for Top Endpoints`:
+
+{{< img src="profiler/guide-monolithic-outliers/3-outliers-monolith-cpu-avg-time-per-call.png" alt="" style="width:100%;" >}}
+
+Now you can see that there is an intermittent spike in CPU utilization where _each call_ to `GET /store_history` takes on average three seconds of CPU time, rather than the increase being because of more calls.
+
+So, the CPU utilization spikes are not due to an increase in traffic but rather an increase in CPU usage per request. 
+
+## Isolate the impact of endpoints
+
+To figure out what is causing increased CPU usage each time `GET /store_history` is called, look at the profiling flame graph for this endpoint during one of the spikes. Select a time range where `GET /store_history` is showing more CPU utilization, scoping the profiling page to that time range. Then switch to the **Flame Graph** visualization to see the methods using the CPU at this time:
+
+{{< img src="profiler/guide-monolithic-outliers/4-outliers-monolith-flame-graph.png" alt="Your image description" style="width:100%;" >}}
+
+To better understand why the `GET /store_history` endpoint in particular is using more CPU, look at the table on the right-hand side, where you can see the endpoint is second from the top. Select just that row to focus the flame graph specifically on the CPU utilization caused by the `GET /store_history` endpoint. 
+
+Because you are investigating resource usage per request, also change the value in the drop-down menu at top of the table to `CPU Time per Endpoint Call`, to show the average resource usage per call to that endpoint instead of the average resource usage per minute.
```

Comment:
```suggestion
Because you are investigating resource usage per request, also change the value in the dropdown at top of the table to `CPU Time per Endpoint Call`. This shows the average resource usage per call to that endpoint instead of the average resource usage per minute.
```